src/components/LoansByCategory/QuickFilters/LocationSelector.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<div class="tw-relative tw-flex tw-flex-col tw-w-full">
+	<div class="tw-relative tw-flex tw-flex-col tw-w-full" v-click-outside="toggleRegions">
```

Comment:
In order for `v-click-outside` to work, the component needs the `clickOutside` mixin. See the component `MonthlyGoodSelectorDesktop` for an example usage.

Reply:
thank you!! I was looking into this and that makes sense now